spec/lib/task_helpers/imports/roles_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe TaskHelpers::Imports::Roles do
+  let(:data_dir)        { File.join(File.expand_path(__dir__), 'data', 'roles') }
+  let(:role_file)       { 'Role_Import_Test.yaml' }
+  let(:bad_role_file)   { 'Bad_Role_Import_Test.yml' }
+  let(:role_one_name)   { 'Role Import Test' }
+  let(:role_two_name)   { 'Role Import Test 2' }
+
+  EvmSpecHelper.seed_specific_product_features(%w(
```

Comment:
I would be willing to be that this is causing your spec failures. This should be in a `before` block.